general/1_terminal/task3-finiak.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+read -p  'enter text: ' name
+
+day=$(echo $name | awk '{print $NF}')
+NUMBER=$(echo $name | tr -dc '0-9')
+
+if [ $day == 'Sunday' ] || [ $day == 'Saturday' ]
+then 
+	echo "Looks like $NUMBER is a weekend day" 
+elif [ $day == 'Monday' ] || [ $day == 'Tuesday' ] || [ $day == 'Wednesday' ] || [ $day == 'Thursday' ] || [ $day == 'Friday' ]  
```

Comment:
Let's make the script more portable and ensure that it works even if the `date` command returns text in, say, Russian, Ukrainan or Indian.